### PR TITLE
Update example to get public keys

### PR DESCRIPTION
**Pull request by artonge**
Signed-off-by: Louis <6653109+artonge@users.noreply.github.com>

### CODE REVIEW

File: doc/api.md [hunk 1]
```diff
@@ -255,7 +255,19 @@ users: Json encoded list of users for which the server should return the public
 
 **Example curl call:**
 
-`curl -X GET https://<user>:<password>@<nextcloud>/ocs/v2.php/apps/end_to_end_encryption/api/v1/public-key -H "OCS-APIRequest:true"`
+```shell
+# Fetch the public key of the loged in user.
```
Comment (tobiasKaminsky) [inline doc/api.md:259]:
```suggestion
# Fetch the public key of the logged in user.
```

File: doc/api.md [hunk 2]
```diff
@@ -255,7 +255,19 @@ users: Json encoded list of users for which the server should return the public
...
+curl \
+	--header 'OCS-APIRequest: true' \
+	https://<user>:<password>@<nextcloud>/ocs/v2.php/apps/end_to_end_encryption/api/v1/public-key
+
+# Fetch the public keys of a list of users.
+curl \
+	--header 'OCS-APIRequest: true' \
+	--data users='["alice","bob"]' \
```
Comment (tobiasKaminsky) [inline doc/api.md:267]: Not sure, but user agent is missing in this example.
Or is this stated somewhere else?

Comment (artonge) [reply doc/api.md]: The user agent is never set in the other calls, hence I did not include it here.
But I can add it if you think it helps.

Comment (tobiasKaminsky) [reply doc/api.md]: Then it is not needed, I think.

**Review by tobiasKaminsky — APPROVED** [review]

**Pull request merged**